sim/ngspice/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# Identify OS
+UNAME_OUT="$(uname -s)"
+case "${UNAME_OUT}" in
+    Linux*)     OS=Linux;;
+    *)          OS="${UNAME_OUT}"
+                echo "Unknown OS: ${OS}"
+                exit;;
+esac
+
```

Comment:
Setting the `OS` variable isn't needed for Conda so please remove unless this is needed specifically for ngspice.